youtube_dl/extractor/mtv.py

```diff
@@ -158,6 +158,12 @@ def _get_video_info(self, itemdoc):
         if mtvn_id_node is not None:
             mtvn_id = mtvn_id_node.text
 
+        content_el = itemdoc.find('.//{http://search.yahoo.com/mrss/}content')
+        if content_el is not None:
+            duration = float(content_el.attrib['duration'])
+        else:
+            duration = None
```

Comment:
Instead of `float` or instead of the whole if? In case of the latter doesn't looking for the attribute raise an error?